Dockerfile.canary

```
@@ -0,0 +1,13 @@
+FROM node:20-bookworm
+
+WORKDIR /src
+
+COPY ../ ./
+RUN npm ci
+RUN npm run build
+
+WORKDIR .
+
+RUN npm run playwright:install
+
+CMD ["npm", "run", "playwright:test:canary"]
```

Can you add a trailing newline here?

```suggestion
CMD ["npm", "run", "playwright:test:canary"]

```